src/main/java/net/masterthought/cucumber/json/Match.java

```diff
@@ -1,13 +1,23 @@
 package net.masterthought.cucumber.json;
 
+import com.google.gson.JsonElement;
+
 public class Match {
 
     // Start: attributes from JSON file report
-    private final String location = null;
+    private final JsonElement location = null;
     // End: attributes from JSON file report
 
     public String getLocation() {
-        return location;
+        if (location.isJsonPrimitive() && location.getAsJsonPrimitive().isString()) {
```

Comment:
ok, now if location is null in feature report there will be 'After' element without location and it isn't included in step statistics.